orgs.js

```diff
@@ -133,6 +133,264 @@
         "type": "org"},
         {"name": "strongloop",
```

Comment:
you need to remove the strongloop org if we are going to enumerate the repos or we'll be doing double accounting;-)